PF/Assignment2/ReverseRightTringle.java

```diff
@@ -0,0 +1,37 @@
+public class ReverseRightTringle {
```

Comment:
ACTION: follow the comments for the class 'PascalTringle' and implement here too.